### PR TITLE
Fix description of get_access() 

### PR DESCRIPTION
**Pull request by VerenaBeckham**
Allow the target of the accessor to be target::host_task.
Based on a comment by @AerialMantis.

### CODE REVIEW

File: adoc/chapters/programming_interface.adoc [hunk 1]
```diff
@@ -4689,7 +4689,8 @@ accessor<T, Dimensions, Mode, Targ> get_access(handler& commandGroupHandler,
       mode and target in the command group buffer.  The accessor is a
       <<ranged-accessor>>, where the range starts at the given offset from the
       beginning of the buffer.  The value of target can be
-      [code]#target::device# or [code]#target::constant_buffer#.
+      [code]#target::device#, [code]#target::constant_buffer# or
+      [code]#target::host_task.
```
Comment (gmlueck) [inline adoc/chapters/programming_interface.adoc:4693]: This is a good catch, but I think we also need to update the description of the overload above (`get_access(handler& commandGroupHandler)`) with the same wording.

Comment (VerenaBeckham) [reply adoc/chapters/programming_interface.adoc:4694]: Ah yes, I missed that. I've now modified that overload's description as well.

**Review by gmlueck — APPROVED** [review]

**Review by keryell — APPROVED** [review]
Thanks.

**Review by AerialMantis — APPROVED** [review]
Looks good, thanks!

**Review by TApplencourt — APPROVED** [review]

**Comment by tomdeakin** [comment]
WG approved to merge.

**Pull request merged**